tap_hubspot/streams.py

```diff
@@ -93,3 +88,43 @@ def schema(self) -> dict:
         if self.cached_schema is None:
             self.cached_schema, self.properties = self.get_custom_schema()
         return self.cached_schema
+
+class PropertiesStream(HubspotStream):
+    """Define custom stream."""
+    schema_filepath = SCHEMAS_DIR / "properties.json"
+    primary_keys = ["name"]
+    replication_key = "updatedAt"
+
+    def parse_response(self, response: requests.Response) -> Iterable[dict]:
+        data = response.json()['results']
+        ret = []
+        for e in data:
+            if self.replication_key not in e:
+                e[self.replication_key] = datetime.datetime.now(tz=utc)
```

Comment:
I guess that works as a workaround 🤷‍♂️

Reply:
Hahaha I decided that I was going for "as long as it works" :laughing: